snp_motion_planning/src/planner_profiles.hpp

```diff
@@ -133,8 +134,27 @@ tesseract_planning::OMPLDefaultPlanProfile::Ptr createOMPLProfile(
 std::shared_ptr<tesseract_planning::TrajOptPlanProfile> createTrajOptToolZFreePlanProfile()
 {
   auto profile = std::make_shared<tesseract_planning::TrajOptDefaultPlanProfile>();
-  profile->cartesian_coeff = Eigen::VectorXd::Constant(6, 1, 5.0);
-  profile->cartesian_coeff(5) = 0.0;
+  Eigen::VectorXd cart_coeff = Eigen::VectorXd::Constant(6, 1, 2.5);
+  cart_coeff(5) = 0.0;  // Set yaw to no cost
+  Eigen::VectorXd cart_tolerance = Eigen::VectorXd::Zero(6);
+  cart_tolerance << 0.01, 0.01, 0.01, 0.05, 0.05, 0.05;  // x, y, z, roll, pitch, yaw
+
+  profile->cartesian_cost_config.enabled = true;
+  profile->cartesian_cost_config.use_tolerance_override = false;
```

Comment:
Why not?

Reply:
I guess it would make sense to set that to true and the tolerances to be all zeros